Demo/src/main/java/com/braintreepayments/demo/CardFragment.java

```diff
@@ -389,15 +389,15 @@ private ThreeDSecureRequest threeDSecureRequest(PaymentMethodNonce paymentMethod
         ThreeDSecureAdditionalInformation additionalInformation = new ThreeDSecureAdditionalInformation()
                 .accountId("account-id");
 
-        ToolbarCustomization toolbarCustomization = new ToolbarCustomization();
-        toolbarCustomization.setHeaderText("Braintree 3DS Checkout");
-        toolbarCustomization.setBackgroundColor("#FF5A5F");
-        toolbarCustomization.setButtonText("Close");
-        toolbarCustomization.setTextColor("#222222");
-        toolbarCustomization.setTextFontSize(18);
+        ThreeDSecureV2ToolbarCustomization toolbarCustomization = (ThreeDSecureV2ToolbarCustomization) new ThreeDSecureV2ToolbarCustomization()
```

Comment:
```suggestion
        ThreeDSecureV2ToolbarCustomization toolbarCustomization = new ThreeDSecureV2ToolbarCustomization()
```
I think that can be removed?

Reply:
This casting was necessary when we were following the builder/method chaining pattern with a base class, so we decided to rethink the design of these classes and move away from the builder pattern. Abandoning the builder pattern in the SDK will make it more Kotlin-friendly moving forward. We updated the v1 and v2 UI customization classes to follow the standard Java getter/setter pattern in this PR, and we plan to update the other builder pattern classes in future PRs.

Reply:
That makes sense 👍